src/dashboard/components/add_dashboard_modal.py

```diff
@@ -0,0 +1,141 @@
+"""Modal for add dashboard buttons."""
+from dash import dcc, html
+
+from dashboard.components import modal
+from dashboard.components.button import button
+
+
+def input(id: str, title: str, description: str) -> html.Div:
+    """Pre-styled input field.
+
+    Args:
+        id (str): Input id
+        title (str): input title
+        description (str): placeholder of the input
+
+    Returns:
+        html.Div: A pre-styled input with a title and placeholder
+    """
+    return html.Div(
+        className="flex flex-col",
+        children=[
+            html.P(title, className=""),
+            dcc.Input(
+                id=id,
+                className="p-3 rounded-md shadow-inner bg-background",
+                placeholder=description,
+            ),
+        ],
+    )
+
+
+def multiline_input(id: str, title: str, description: str) -> html.Div:
+    """Pre-styled multiline input.
+
+    Args:
+        id (str): id of the multiline input
+        title (str): title of the multiline input
+        description (str): placeholder of the multiline input
+
+    Returns:
+        html.Div: pre-styled multiline input with title and placeholder
+    """
+    return html.Div(
+        className="flex flex-col",
+        children=[
+            html.P(title, className=""),
+            dcc.Textarea(
+                id=id,
+                draggable=False,
+                className="p-3 rounded-md shadow-inner h-[17rem] bg-background",
+                placeholder=description,
+            ),
+        ],
+    )
+
+
+def inputs() -> html.Div:
+    """Input container.
+
+    Returns:
+        html.Div: Div containing the text inputs
+    """
+    return html.Div(
+        className="flex flex-col px-[5rem] py-5 space-y-4",
+        children=[
+            html.P("Create dashboard", className="text-3xl"),
+            input("dashboard-title", "Dashboard title", "Enter dashboard title..."),
+            multiline_input("dashboard-desc", "Description", "Enter dashboard description..."),
+        ],
+    )
+
+
+def buttons() -> html.Div:
```

Comment:
Consider renaming buttons() and inputs() to generate_buttons() etc. for consistency.